components/applications-service/dev/hab-client/run-hab-sup.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+main() {
+  token=""
+
+  # Parse command line flags and options.
+  while getopts ":ht:" opt; do
+    case "${opt}" in
+      h)
+        print_help
+        exit 0
+        ;;
+      t)
+        token="${OPTARG}"
```

Comment:
nit: should we check when the provided token (that is the flag) is empty?

Reply:
added!